src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/jsonb/JsonbDiagnosticsCollector.java

```diff
@@ -87,7 +88,7 @@ private void collectJsonbTransientFieldDiagnostics(PsiJavaFile unit, PsiClass ty
                 List<String> jsonbAnnotationsForAccessor = getJsonbAnnotationNames(type, accessor);
                 if (hasJsonbAnnotationOtherThanTransient(jsonbAnnotationsForAccessor)) {
                     createJsonbTransientDiagnostic(unit, diagnostics, accessor, jsonbAnnotationsForAccessor,
-                            JsonbConstants.ERROR_MESSAGE_JSONB_TRANSIENT_ON_FIELD);
+                            JsonbConstants.DIAGNOSTIC_CODE_ANNOTATION_TRANSIENT_FIELD);
```

Comment:
should this be `DIAGNOSTIC_CODE_ANNOTATION_TRANSIENT_ACCESSOR`? Just going by the other params, `accessor` and `jsonbAnnotationsForAccessor`, and the usage of it below.

Reply:
No, the diagnostic is on the accessor but it is flagging a problem caused by the field. So there is also a message on the field where the problem arises.